components/SingleChatHeader.js

```diff
@@ -78,12 +78,12 @@ const SingleChatHeaderCenter = (props) => {
 
 const mapState = (state) => {
   const chat = state.chats.currentChat
-  const getMemberNames = chat
-    ? memberNameHelper(Object.values(chat.members))
-    : []
-  const getMemberImgs = chat
-    ? memberImgHelper(Object.keys(chat.members), state.user.contacts)
-    : []
+  const getMemberNames =
+    chat && chat.members ? memberNameHelper(Object.values(chat.members)) : []
+  const getMemberImgs =
+    chat && chat.members
+      ? memberImgHelper(Object.keys(chat.members), state.user.contacts)
+      : []
   return {
```

Comment:
just curious if you could provide some context for this change. i changed this variable as well and I'll need to merge it with my changes.

Reply:
before when i just had chat.members it would error out when switching screens so i just put the first chat as a check so it won't error out

Reply:
but if it works for you without it feel free to take it out